### DISCUSSION TITLE
[Question]: FFMPEG running in background with everything disabled

### DISCUSSION THREAD

**Discussion by luisiam** [blakeblackshear/frigate#10927] [Ask A Question] [answered]
### What is your question:

Even if I disabled everything, birdseye, motion, detect, record, snapshot, audio, ffmpeg is still running. Is it possible to stop the ffmpeg process if nothing is enabled?

![image](https://github.com/blakeblackshear/frigate/assets/16310467/615f85e0-f0e6-4ab5-a2c8-78eafe32708b)

**Comment by NickM-27** [comment, marked as answer]
no, currently the stream is always decoded as it is used for the webui, motion detection, etc
